tests/unit/web/test_app.py

```diff
@@ -64,6 +64,7 @@ def test_config(client: FlaskClient):
         "wait_time": [10, 20],
         "mav": [3, ""],
         "volume": [1, ""],
+        "avg_buy_price": ["", 100],
```

Comment:
What do you mean by "daily profit"? Do you mean the relative change over the displayed range? The thing in red in the img below? 

![IMG_1695](https://github.com/loiccoyle/tinyticker/assets/33181239/2baa96f8-2c6d-48b6-99a7-b55cecc50cb0)


>  why replace the daily profit with avg?

Because there isn't enough space, unless I add it small here instead?

![IMG_1695-2](https://github.com/loiccoyle/tinyticker/assets/33181239/a5e199bc-f1bc-4ded-bad7-01eab0c120a6)


> Also I would add the word "daily" or "1 day" before the regular profit percentage and the word "profit" before the avg percentage.

I think what you call the "regular profit percentage" is not daily but over the current displayed range, so if it is showing the last 30 days then it is over the last 30days.  And there really isn't enough space to display any words.

Reply:
I cant see the last two pictures you attached here.

Reply:
Sorry, fixed

Reply:
I think what you suggest is a good option.
`relative changes` one line lower and the `avg_buy_price` when is set will where `relative changes` was. 
I guess there will be no space left to add the word `profit` before the `avg_buy_price` profit place ?

Reply:
Ok I'll go with that then, no there is no space for the word `profit` also I would not call it profit as it can very well be a loss... But in any case there is no space.

Reply:
ok  resolved in https://github.com/loiccoyle/tinyticker/pull/44/commits/e88f57dff001cf1d436347d057412dc5cefea17e, it now looks like: 
![IMG_1696](https://github.com/loiccoyle/tinyticker/assets/33181239/665dc180-54f4-4bf9-a0a4-459d9daa47eb)

Am I good to merge?

Reply:
Excellent !